src/reconcile/rec_row.c

```diff
@@ -609,7 +609,8 @@ __rec_row_garbage_collect_fixup_update_list(WT_SESSION_IMPL *session, WT_RECONCI
     if (first_upd->type == WT_UPDATE_TOMBSTONE)
         return (0);
 
-    if (WT_TXNID_LT(first_upd->txnid, btree->oldest_live_txnid)) {
```

Comment:
I also think that transaction IDs are relevant for this decision making. This is about content in the ingest table, which is local to this node. So transaction IDs are relevant to that decision making.

Reply:
The data in the stable table is with transaction ids from a different node. From transaction ids, you cannot tell if the particular updates is included in the stable table or not.

Reply:
I think the only thing we need to do here for transaction ids is to ensure we don't delete uncommitted updates. I'll fix this.